CameraExtrinsicMat:
   rows: 4
   cols: 4
   dt: d
   data: [-3.99813366e-02,  1.99986667e-02,  9.99000273e-01, 7.1070600895280145e-01,
          -9.99200107e-01,  2.16623653e-19, -3.99893342e-02, -2.3349739393366022e-02,
          -7.99733366e-04, -9.99800007e-01,  1.99826699e-02, -3.1815435704534178e-01,
                       0.,              0.,              0.,                       1.]
CameraMat:
   rows: 3
   cols: 3
   dt: d
   data: [ 1.3418542631706748e+03, 0., 3.5844774941989658e+02, 0.,
       1.3633533700157413e+03, 2.2298273098324543e+02, 0., 0., 1. ]
DistCoeff:
   rows: 1
   cols: 5
   dt: d
   data: [ 9.7388426101958039e-02, -1.4798811810985903e+00,
       -1.8136642223375680e-02, -6.6065762144425187e-03,
       4.5606590298897780e+00 ]
ImageSize: [ 800, 600 ]
ReprojectionError: 9.9056821055071453e-01
